### ISSUE TITLE
[Site] Helper classes

### ISSUE THREAD

**Issue by hammy2899** [hammy2899/Fomantic-UI#22]
Create helper classes for padding, margin and may be responsive breakpoints.

Original Issue: https://github.com/Semantic-Org/Semantic-UI/issues/6195

Examples:
```
.large.top.margin
.small.left.padding
.only.show.on.mobile
.only.show.on.computer
.dont.show.on.tablet
only.show.on.tablet.and.mobile
```
etc.

**Comment by hammy2899** [comment]
Going to put this on hold for now until the device visibility is more thought out.

**Comment by lubber-de** [comment]
I am using the css additions from https://github.com/Semantic-Org/Semantic-UI/issues/1114#issuecomment-201057438 which adds global support for 

mobile only
mobile hidden,
tablet only
tablet hidden
computer only
computer hidden
large screen only
large screen hidden
widescreen only
widescreen hidden
or lower hidden

 and works flawlessly so far

**Comment by hammy2899** [comment]
@lubber-de That could be a starting point

**Comment by etshy** [comment]
Is it still on hold ?

It could be good to have these kind of helpers directly inside the library

**Comment by hammy2899** [comment]
@etshy This is on hold for now because we want to do more then just these modifiers and we also want to make it customizable as much as possible allow users to put these classes on anything and different combinations and for it to still work. This will take a lot of testing/work which is why it is on hold.

**Comment by etshy** [comment]
oooh, nice.

Thanks for all that work. 👍

**Comment by ivantcholakov** [comment]
https://github.com/Semantic-Org/Semantic-UI/issues/6195#issuecomment-465525195

Something for margins/padding would be useful. Because I hate my buttons to stick vertically like that:

![image](https://user-images.githubusercontent.com/1438104/105649244-125cd800-5eb8-11eb-9439-5e3228791aa2.png)

**Comment by lubber-de** [comment]
Button groups now have a `spaced` variant which takes care about a margin next to lower buttons
https://fomantic-ui.com/elements/button.html#spaced-buttons

**Comment by TheDevelolper** [comment]
Really need this in 2023!

**Comment by lubber-de** [comment]
@TheDevelolper Look at my comment above.
https://github.com/fomantic/Fomantic-UI/issues/22#issuecomment-429109342

**Comment by TheDevelolper** [comment]
> @TheDevelolper Look at my comment above.
> https://github.com/fomantic/Fomantic-UI/issues/22#issuecomment-429109342

Yeah I saw this but I'm using formantic via npm so not sure if it's possible to tell npm to use that branch? 

I added my own mobile only class for the moment.

**Comment by lubber-de** [comment]
Which Branch? My comment points to a post which contains some additional custom css which you can just copy into your own project.
